wolfssl/ssl.h

```diff
@@ -639,6 +639,15 @@ struct WOLFSSL_X509_STORE_CTX {
 
 typedef char* WOLFSSL_STRING;
 
+typedef struct WOLFSSL_RAND_METHOD {
+    int (*seed)(const void* seed, int len);
+    int (*bytes)(unsigned char* buf, int num);
```

Comment:
Can we have these callbacks use consistent `len` vs `num`. I prefer `len`.

Reply:
Updated, but I should note the parameter names were taken from our wolfSSL_RAND functions, so the naming is inconsistent there too.  Should I change their parameter names as well?